rpcserver.go

```diff
@@ -3001,9 +3028,44 @@ func (r *rpcServer) QueryProof(ctx context.Context,
 		"(universeID=%x, leafKey=%x)", universeID,
 		leafKey.UniverseKey())
 
+	// Keep a record of whether the proof type was specified by the client.
+	unspecifiedArgProofType :=
+		universeID.ProofType == universe.ProofTypeUnspecified
```

Comment:
Can use a parens to avoid the awkward line break here: 
```go
unspecifiedArgProofType := (universeID.ProofType == 
    universe.ProofTypeUnspecified
)
```

Reply:
I get a `syntax error: unexpected newline, expected )` error with the syntax you've suggested.